### PR TITLE
add readonly oracle contract and test

### PR DESCRIPTION
**Pull request by Craigson**

### CODE REVIEW

File: contracts/modules/oracle/ReadonlyOracle.sol [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    constructor(address _registry) {
+        registry = FeedRegistryInterface(_registry);
+    }
+
+    /**
+     * @param token - ERC20 token to get USD price for
+     * @return price - the latest price in USD to 8 decimals
+     */
+    function getLatestAnswer(address token) external view returns (int256) {
```
Comment (Craigson) [inline contracts/modules/oracle/ReadonlyOracle.sol:33]: @kibagateaux we cannot call it `_getLAtestAnswer` as it differs from the `IOracle` function name. We can either omit inheriting from `IOracle` or leave it the same. Up to you.

Comment (kibagateaux) [reply contracts/modules/oracle/ReadonlyOracle.sol:33]: Ah i wanted the function in the same oracle contract. U can just copypasta this code over. Its just their for cheaper/easier reads onchain + offchain if someone doesnt care about all the additional security checks we do

File: contracts/modules/oracle/Oracle.sol [hunk 2]
```diff
@@ -70,4 +70,37 @@ contract Oracle is IOracle {
             return NULL_PRICE;
         }
     }
+
+    function _getLatestAnswer(address token) external view returns (int256) {
```
Comment (kibagateaux) [inline contracts/modules/oracle/Oracle.sol:74]: Add comment "view function for use offchain. Can be used onchain for less gas than `getLatestAnswer` with security tradeoffs"

Comment (Craigson) [reply contracts/modules/oracle/Oracle.sol:80]: done

**Pull request merged**